app/src/main/res/layout/fragment_mentor.xml

```diff
@@ -0,0 +1,77 @@
+    android:id="@+id/main"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <Button
+        android:id="@+id/fav_button"
+        android:layout_width="117dp"
+        android:layout_height="34dp"
+        android:layout_marginTop="52dp"
+        android:layout_marginEnd="27dp"
+        android:background="@drawable/favorite_project_button"
+        android:text="@string/fav_button_text"
+        android:textColor="#222B5C"
+        android:textSize="15dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/mentor_header"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="27dp"
+        android:layout_marginTop="54dp"
+        android:layout_marginEnd="129dp"
+        android:text="@string/mentor_header"
+        android:textColor="#222B5C"
+        android:textSize="20dp"
+        app:layout_constraintEnd_toStartOf="@+id/fav_button"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/mentor_recycler"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="6dp"
+        android:layout_marginTop="180dp"
+        android:layout_marginEnd="6dp"
+        android:background="@android:color/transparent"
+        android:padding="10dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <com.google.android.material.tabs.TabLayout
+        android:id="@+id/projectCategories"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="23dp"
+        app:tabIndicatorColor="#E35058"
+        app:tabSelectedTextColor="#E35058"
+        app:tabTextColor="#222B5C"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/fav_button">
+
+        <com.google.android.material.tabs.TabItem
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/web_dev_project" />
+
+        <com.google.android.material.tabs.TabItem
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/language_project" />
+
+        <com.google.android.material.tabs.TabItem
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text= "@string/systems_project" />
+    </com.google.android.material.tabs.TabLayout>
```

Comment:
Could you add the horizontal blue bar all the way across the screen, like in Figma?

Here's how I did it for the MapsFragment: https://github.com/HackIllinois/android/blob/maps-fragment/app/src/main/res/layout/fragment_maps.xml#L90.

You just add a View with height 1dp and constrain it to the bottom of the TabLayout.